GEOSldas_App/GEOSldas_HISTdet.rc

```diff
@@ -0,0 +1,70 @@
+#
+#  Sample GEOSldas HISTORY.rc file for LADAS (central simulation)
+#
+#  This sample is for the GEOSldas instance that is coupled with the central
+#  simulation component of the Hy4dEnVar ADAS:
+#
+#  (1)  The "catch_progn_incr" is output is the ensemble average. 
+#  (2)  The "catch_progn_incr" output is in tile space.  Its definition is generic
+#
+##################################################################################
+
+VERSION: 1
+EXPID:   GEOSldas_expid
+
+COLLECTIONS:
```

Comment:
I think we need to add the usual diagnostic `tavg[*]_2d_lnd_Nx`, and `inst[*]_2d_lndfcstana_Nx` collections that we will want to output from the LDAS whenever it's running in the coupled land-atm DA system.  

The time step could be hourly (as in FP and reanalysis) or perhaps 3-hourly (as in SMAP L4).  

Note that the aforementioned collection names are from the MERRA-2 era and reflect what is in FP now.   For M21C, there's slightly different naming convention.  I assume the newer convention will be adopted in FP when it transitions to the forthcoming "v12" (181-level) GCM, so maybe it's best to use the newer collection naming convention here.

Reply:
those diag collections can be added to the template.  Because they are lat/lon outputs, more modification is needed in setup, we will need to insert the grid name ( from setup info) in the specification of "regrid_name", probably a corresponding "grid_label"  as well.